pages/operations/pricing-calculator.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Pricing Calculator
+description: >-
+  Estimate the costs of using Plural to running the applications
+  and configuration you need for your business with our Pricing Calculator.
+---
+
+{% calculator %}
+{% /calculator %}
+
+## How do we calculate these costs?
+
+Plural deploys a standard configuration on each cloud provider, and uses provider-specific price lists to estimate an infrastructure cost for the deployment of your applications.
+
+{% callout severity="info" title="How do we calculate cloud costs?" %}
+Lorem ipsum dolor sit amet consectutor. Lorem ipsum dolor sit amet consectutor. Lorem ipsum dolor.
```

Comment:
Let's move the above text under this callout. So this would then be:
"Plural deploys a standard configuration on each cloud provider and uses provider-specific price lists to estimate an infrastructure cost for the deployment of your applications. Each initial deployment has a certain amount of headroom for installing applications and will scale accordingly as more are added. Costs to Plural are calculated based on which plan is chosen; Plural Professional is priced by the number of users and clusters."